src/DotNetBridge/Entrypoints.cs

```diff
@@ -87,5 +87,59 @@ public static CommonOutputs.TransformOutput CreateVariableColumn(IHostEnvironmen
             var xf = VariableColumnTransform.Create(env, inputOptions, inputOptions.Data);
             return new CommonOutputs.TransformOutput { Model = new TransformModelImpl(env, xf, inputOptions.Data), OutputData = xf };
         }
+
+        public sealed class Input
```

Comment:
Is this name to general since it exists in a file that contains more than one type of Input/Output class?